scripts/ci/jobs/gke_version_compatibility_tests.py

```diff
@@ -32,22 +31,18 @@
 # set required test parameters
 os.environ["ORCHESTRATOR_FLAVOR"] = "k8s"
 
-central_chart_versions = get_latest_helm_chart_versions(
-    "stackrox-central-services", 2)
-sensor_chart_versions = get_latest_helm_chart_versions(
-    "stackrox-secured-cluster-services", 3
-)
+central_chart_versions, sensor_chart_versions = get_supported_helm_chart_versions()
+
 makefile_path = Path(__file__).parent.parent.parent.parent
 latest_tag = subprocess.check_output(
     ["make", "tag", "-C", makefile_path, "--quiet", "--no-print-director"],
     shell=False,
     encoding="utf-8",
 ).strip()
 
-if len(central_chart_versions) == 0:
-    raise RuntimeError("Could not find central chart versions.")
-if len(sensor_chart_versions) == 0:
-    raise RuntimeError("Could not find sensor chart versions.")
+if len(central_chart_versions) == 0 and len(sensor_chart_versions) == 0:
+    logging.info("Found no older versions to test against according to the product lifecycles API. \n"
+                 "However versions with support exceptions will still be tested against.")
```

Comment:
[nit] I kind of like having the `ifs` separated like they were. That way if for whatever reason there's a change in the way we retrieve the charts and we land in a situation where we have charts for central but not for sensor (or the other way around) we can log exactly what is going on.

Reply:
The reason I changed this is that with the product lifecycles API support for versions between central and sensor is now always symmetrical. 

> That way if for whatever reason there's a change in the way we retrieve the charts and we land in a situation where we have charts for central but not for sensor (or the other way around) we can log exactly what is going on.

This should already be logged in `__get_supported_helm_chart_versions` where it prints a `debug.logging` message for each central or sensor chart that is supported but not available in helm

Reply:
> is now always symmetrical

My point exactly, `now` is symmetrical but it might not be at some point. I think my initial point was confusing because I wrote `the way we retrieve the charts` when I meant `the way we retrieve the supported versions`.